lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -0,0 +1,293 @@
+module GitHubChangelogGenerator
+  # A Fetcher responsible for all requests to GitHub and all basic manipulation with related data
+  # (such as filtering, validating, e.t.c)
+  #
+  # Example:
+  # fetcher = GitHubChangelogGenerator::OctoFetcher.new options
+
+  class OctoFetcher
+    PER_PAGE_NUMBER   = 100
```

Comment:
Changed this to use the maximum PER_PAGE_NUMBER. This helps with large repositories, where we need to minimize API calls